serverless-test-project/package.json

```diff
@@ -3,22 +3,22 @@
   "version": "2.1.2",
   "description": "Test project for SLIC Watch serverless plugin",
   "main": "basic-handler.js",
+  "type": "module",
   "repository": {
     "type": "git",
     "url": "https://github.com/fourTheorem/slic-watch.git",
     "directory": "serverless-test-project"
   },
   "license": "Apache",
   "devDependencies": {
-    "ajv": "^8.11.0",
-    "serverless": "^3",
-    "serverless-step-functions": "^3.7.0"
+    "serverless-step-functions": "^3.12.1"
   },
   "scripts": {
     "test": "ALARM_TOPIC=test sls package",
     "test:v2": "ALARM_TOPIC=test sls package -c serverless-v2.yml"
   },
   "dependencies": {
-    "axios": "^0.27.2"
+    "axios": "^1.3.4",
+    "serverless": "^2.72.3"
```

Comment:
This should be a `devDependency` - there shouldn't be a need to package as a production dependency